### ISSUE TITLE
Color contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by alomarramaa** [samvera/hyku#3092]
https://vpat.hykudemo.org/ and https://vpat.hykudemo.org/concern/etds/9589c183-af3e-440f-8d1d-4e0c96a35a67

Version 7. 1

### Descriptive Summary
Color contrast requirements ensure that text is readable against its background by meeting minimum contrast ratios defined by WCAG. Insufficient contrast can make content difficult or impossible to read, especially for users with visual impairments. WCAG AA criteria require a minimum contrast ratio of at least 4.5:1 between small text and background.

### Expected Behavior
Increase color contrast to at least 4:5:1 for the following issues.

### Steps to Reproduce 
1. Head to https://vpat.hykudemo.org/ for issues pertaining to home page theme accessibility concerns, or https://vpat.hykudemo.org/concern/etds/9589c183-af3e-440f-8d1d-4e0c96a35a67 for issues regarding show page theme concerns.
2. Use SiteImprove's browser extension to scan and find issue

### Occurrences of Issue

1. Institutional Repository Home Page
Increase Home, About, Help and Contact all to 4.5:1 or higher. They are all currently 3.94 : 1
<img width="564" height="336" alt="Image" src="https://github.com/user-attachments/assets/5151a64f-f690-4a0f-abf3-13b11cf7dca7" />

2. Default Show Page at https://vpat.hykudemo.org/concern/etds/9589c183-af3e-440f-8d1d-4e0c96a35a67
Increase the color contrast to 4.5:1 or higher.
Current Contrast ratio: 4.42 : 1

<img width="565" height="333" alt="Image" src="https://github.com/user-attachments/assets/b29072d3-1e20-4b74-9e0b-7506abbd5094" />

3. Cultural Show Page at https://vpat.hykudemo.org/concern/etds/9589c183-af3e-440f-8d1d-4e0c96a35a67
Increase the color contrast to 4.5:1 or higher.
Current Contrast ratio: 4.16 : 1 for Home, Dashboard & Works 

<img width="568" height="386" alt="Image" src="https://github.com/user-attachments/assets/ecb8991a-4e95-4ae4-9e85-a2596ef280b8" />
